nixos/roles/webproxy.nix

```diff
@@ -63,9 +63,29 @@ in
           command = "${cfg.package}/bin/varnishadm status";
           timeout = 180;
         };
-        varnish_http = {
-          notification = "varnish port 8008 HTTP response";
-          command = "check_http -H localhost -p 8008 -c 10 -w 3 -t 20 -e HTTP";
+        varnish_http = let
+          # the plattform module passes multiple addresses in the form of '<address>:<port> -a <address>:<port> -a <address>:<port>'
+          first_address = builtins.elemAt (lib.splitString " " cfg.http_address) 0;
```

Comment:
The question here is what we want to achieve with our monitoring:

- monitor the varnish server process and whether it works -> it is sufficient to reach varnish via *any* of its multiple listen addresses, in this implementation we always take the first one
- monitor the listen address configuration as well: is varnish actually reachable via all configured IP-port configurations? Did we mis-type something, did one of the IPs somehow change, etc…
  - in that case, we need to run the http check against __all__ the configured listen addresses, not just the first one.